randovania/games/am2r/json_data/Hydro Station.txt

```diff
@@ -1200,11 +1229,37 @@ Extra - minimap_data: [{'x': 19, 'y': 23}, {'x': 19, 'y': 24}, {'x': 20, 'y': 23
   * Layers: default
   * 4-Tiles High Dock to Breeding Grounds Lobby/Horizontal Dock to Breeding Grounds Alpha Nest West
   > Dock to Breeding Grounds Overgrown Alley
-      Speed Booster and Shinesparking Tricks (Beginner) and Go Through Vines
+      All of the following:
+          Speed Booster and Shinesparking Tricks (Beginner)
+          Any of the following:
+              # Go Through Vines
+              Gravity Suit
+              Damage Boost (Beginner) and Lava Damage ≥ 60
   > Event - Alpha
       All of the following:
           Defeat dodging Alpha
-          Go Through Vines or Hijump With Power Grip Wall
+          Any of the following:
+              # Go Through Vines -  bottom way
+              Gravity Suit or Can Use Any Bombs
+              Any of the following:
+                  # Top way over floating blocks
+                  Space Jump
+                  All of the following:
+                      # Hijump Method
+                      Hi-Jump Boots
+                      Power Grip or Walljump (Advanced)
+                  All of the following:
+                      # Spider Method
+                      Can Use Spider Ball
+                      Mid-Air Morph (Intermediate) or Can Use Bombs or Can Use Spring Ball
+                  # WJ Method
+                  Morph Ball and Morph Glide (Intermediate) and Walljump (Advanced)
+                  # CBJ method
+                  Charged Bomb Jump (Advanced) and Can Use Charged Bomb Jump
+                  All of the following:
+                      # IBJ method
+                      Can Use Bombs
+                      Diagonal Infinite Bomb Jumping (Advanced) or Infinite Bomb Jumping (Advanced)
```

Comment:
Assuming this `or` is intended as you can either go high enough then over with IBJ or direct with DIBJ.

Reply:
correct.